Makefile

```diff
@@ -1,25 +1,177 @@
-# Delegate all make targets directly to ant
+# Makefile for Sphinx documentation
```

Comment:
All these changes are just a general refresh of the sphinx setup? or a move of the lower level to the top level?

Reply:
general refresh